x-pack/plugins/data_visualizer/public/application/common/components/stats_table/components/field_data_row/use_column_chart.tsx

```diff
@@ -107,6 +111,14 @@ export const getLegendText = (chartData: ChartData, maxChartColumns: number): Le
     );
   }
 
+  if (isOrdinalChartData(chartData) && isNumeric) {
+    // The original data could be numerical but also ordinal (e.g. "2340")
+    return i18n.translate('xpack.dataVisualizer.dataGridChart.singleTopValueLegend', {
+      defaultMessage: `{cardinality, plural, one {# top value} other {# top values}}`,
```

Comment:
For numeric fields, with only value, I don't think we need the word 'top' in the text. Just `1 value` would be better I think.

![image](https://user-images.githubusercontent.com/7405507/150839005-52566996-1bbc-4f9a-a2ef-5c5e472c8c3b.png)

Reply:
Updated here https://github.com/elastic/kibana/pull/123306/commits/09fa4366632f71f1d8fbb8fc71afe278fc9dc89d. Also added example value if cardinality is 1.
<img width="1007" alt="Screen Shot 2022-01-24 at 12 56 02" src="https://user-images.githubusercontent.com/43350163/150846254-4a6f7d9a-0780-49c3-bae9-8911098833b3.png">

Reply:
This change works well I think. Here's another example:

![image (8)](https://user-images.githubusercontent.com/7405507/150861681-70e93fa0-0ea2-484f-ba0d-db09c6bff564.png)

Compared to before this PR:

![image (9)](https://user-images.githubusercontent.com/7405507/150861719-d3c990c8-c6fb-4191-8690-d6fd296f70df.png)